jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -36,8 +34,10 @@ def pachyderm_resources():
     yield repos, branches, files
 
     for repo in repos:
-        client.pfs.delete_repo(repo=pfs.Repo(name=repo))
-
+        try:
+            client.pfs.delete_repo(repo=pfs.Repo(name=repo))
```

Comment:
[nit] i don't think delete_repo errors on a nonexistent repo

Reply:
Ah good to know, will remove the try catch

Reply:
Removed this